charts/sherlock/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: sherlock
+version: 0.0.0
```

Comment:
er does version 0.0.0 work or does it need to be 0.1.0 or something non zero

Reply:
It'll get auto-bumped bumped to 0.1.0 once I merge this :)